packages/lib/event.ts

```diff
@@ -111,18 +111,56 @@ export function getEventName(eventNameObj: EventNameObjectType, forAttendeeView
 }
 
 export function updateHostInEventName(eventName: string, oldHost: string, newHost: string) {
-  const oldHostFirstName = oldHost.split(" ")[0];
-  const newHostFirstName = newHost.split(" ")[0];
-
-  if (!eventName.includes(oldHost) && !eventName.includes(oldHostFirstName)) {
-    return eventName;
-  }
+  const oldParts = oldHost.trim().split(/\s+/);
+  const newParts = newHost.trim().split(/\s+/);
+
+  // Handle cases where names might have different number of parts
+  const oldFirst = oldParts[0];
+  const oldLast = oldParts.slice(1).join(" ");
+  const newFirst = newParts[0];
+  const newLast = newParts.slice(1).join(" ");
+
+  const formats = [
+    // Full name patterns (prioritize these first)
+    ...(oldLast
+      ? [
+          {
+            pattern: `${oldFirst}.${oldLast}`,
+            replacement: newLast ? `${newFirst}.${newLast}` : newFirst,
+          },
+          {
+            pattern: `${oldFirst}-${oldLast}`,
+            replacement: newLast ? `${newFirst}-${newLast}` : newFirst,
+          },
+          {
+            pattern: `${oldFirst}_${oldLast}`,
+            replacement: newLast ? `${newFirst}_${newLast}` : newFirst,
+          },
+          {
+            pattern: `${oldFirst} ${oldLast}`,
+            replacement: newLast ? `${newFirst} ${newLast}` : newFirst,
+          },
```

Comment:
Is there a reason we want to preserve this? I would imagine we should always have these in 
`{FirstName} {LastName}` format. 🤔